packages/components/src/components/dropdown/dropdown.tsx

```diff
@@ -487,7 +489,15 @@ const Dropdown = ({
                         id='dropdown-display'
                         ref={dropdown_ref}
                     >
-                        {!!suffix_icon && <Icon className='suffix-icon' icon={suffix_icon} size={16} />}
+                        {!!suffix_icon && (
+                            <Icon
+                                className={classNames('suffix-icon', {
+                                    'suffix-icon--flip': is_list_visible && should_animate_suffix_icon,
+                                })}
+                                icon={suffix_icon}
+                                size={16}
```

Comment:
isn't the default size 16?

Reply:
ah yeah, i didnt add that, but ill remove it 👍